akka-projection-slick/src/test/scala/akka/projection/slick/SlickSpec.scala

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.projection.slick
+
+import akka.actor.ActorSystem
+import akka.projection.testkit.ProjectionTestRunner
+import com.typesafe.config.Config
+import org.scalatest.BeforeAndAfterAll
+import org.scalatest.concurrent.{ Eventually, ScalaFutures }
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpecLike
+import slick.basic.DatabaseConfig
+import slick.jdbc.H2Profile
+
+import scala.concurrent.Await
+import scala.concurrent.duration._
+
+abstract class SlickSpec(config: Config)
+    extends AnyWordSpecLike
+    with Matchers
+    with ScalaFutures
+    with ProjectionTestRunner
+    with BeforeAndAfterAll
+    with Eventually {
+
+  private var _actorSystem: ActorSystem = _
+
+  implicit def actorSystem: ActorSystem = _actorSystem
```

Comment:
What do you think about writing tests with Typed instead?
This could be a `ScalaTestWithActorTestKit`. Some nice things even when not using actors, for example log capturing and LoggingTestKit.

Reply:
I'm using now the Typed TestKit, but keeping the `SlickSpec` name. 

On another PR I will tackle the move to `ScalaTestWithActorTestKit`. I'm planing some improvements in `ProjectionTestRunner` #42, so we can do that in one PR.

Reply:
FYI, I refactored it to be a `ScalaTestWithActorTestKit` as you suggested, but kept the name `SlickSpec`. 

I was planing to postpone it and pick a name more aligned to `ScalaTestWithActorTestKit` while refactoring the `ProjectionTestRunner` but I changed my mind.